triggers/daml/Daml/Trigger/Internal.daml

```diff
@@ -39,8 +39,11 @@ data ACS = ACS
   { activeContracts :
 #ifdef DAML_GENMAP && DAML_GENERIC_COMPARISON
       GMap.Map TemplateTypeRep
+         (GMap.Map
+#else
+      [] ((,)
```

Comment:
This seems a bit too code golfy. I think just moving `AnyContractId AnyTemplate` in both branches would be easier to parse`.

Reply:
Sure. c22f62e8cc